crates/diesel_models/src/payout_attempt.rs

```diff
@@ -30,6 +30,7 @@ pub struct PayoutAttempt {
     pub profile_id: common_utils::id_type::ProfileId,
     pub merchant_connector_id: Option<common_utils::id_type::MerchantConnectorAccountId>,
     pub routing_info: Option<serde_json::Value>,
+    pub additional_payout_method_data: Option<serde_json::Value>,
```

Comment:
Let's make it the optional type - `AdditionalPayoutMethodData`? We will not need to manually encode or decode the values in our flows that way.